### DISCUSSION TITLE
No bluetooth

### DISCUSSION THREAD

**Discussion by vivelespatess** [badstorm/surface-pro-7-opencore#127] [Q&A]
My bluetooth does not work, I did not modify any kext. You will find supporting images.

<img width="1368" alt="Capture-d-cran-2023-06-23-20-37-56" src="https://github.com/badstorm/surface-pro-7-opencore/assets/16374830/bd07e811-2595-4d08-a136-b0ef81b216a4">
<img width="1368" alt="Capture-d-cran-2023-06-23-20-38-36" src="https://github.com/badstorm/surface-pro-7-opencore/assets/16374830/63187789-78a8-4131-bbcf-5204c8259634">

Surface pro 7 
intel core i5 
8gb ram

**Comment by xealstorm** [comment]
You may grab the latest AirportItlwm kext from here: https://github.com/OpenIntelWireless/itlwm/releases 
2.2.0 is stable now
Don't forget to turn it on in the config and turn off the itlwm.

**Comment by vivelespatess** [comment]
I took the latest release of AirportItlwm for Ventura and I also took all the necessary kext from https://github.com/OpenIntelWireless/IntelBluetoothFirmware including:
`IntelBTPatcher.kext` `IntelBluetoothInjector.kext` 

and some from https://github.com/acidanthera/BrcmPatchRAM including: 
`BlueToolFixup.kext` `BrcmPatchRAM3.kext` and `BrcmFirmwareData.kext`

and bluetooth still not working, but wifi is ok. Am I doing something wrong?

![Capture-d-cran-2023-06-24-12-09-30](https://github.com/badstorm/surface-pro-7-opencore/assets/16374830/72aee2bd-9ef5-4cb6-911f-6ddf59bf33ec)

**Comment by Buggie105** [comment]
I got Bluetooth working on my Surface Pro 7 by updating just BlueToolFixup.kext to 2.6.8
link: [BrcmPatchRAM 2.6.8](https://github.com/acidanthera/BrcmPatchRAM/releases/tag/2.6.8)

![Screenshot 2023-10-07 at 3 03 29 PM](https://github.com/badstorm/surface-pro-7-opencore/assets/40400933/0da51926-d264-4a6a-b59c-f3b5619d310c)

**Comment by dzaikos** [comment]
> I got Bluetooth working on my Surface Pro 7 by updating just BlueToolFixup.kext to 2.6.8 link: BrcmPatchRAM

I updated this kext and was able to at least turn Bluetooth on, but I couldn't pair any new devices. I tried a Logitech MX Anywhere 3S and an Xbox Controller. Every time it just tries to pair but silently times out.

With just this change are you able to pair new devices?

**Comment by badstorm** [comment]
Since the conversation is dated I am closing the issue and moving it as a conversation.
